### DISCUSSION TITLE
Debug with built in "simple browser"

### DISCUSSION THREAD

**Discussion by benjamin-fukdawurld** [coder/code-server#5527] [Ideas]
Hey guys, I'm using vscode to develop web app and I was wondering how could I launch my app In simple browser and attach the debugger to it.

If I could launch the app in the built in browser attach the debugger and use hmr all combined, this would be the ultimate live web dev setup.

Maybe it's not yet feasible but if it is could some one please help me or send me to some doc that could help me figure this out.

**Comment by jsjoeio** [comment]
Hmm..🤔 I haven't had much luck attaching the debugger to a web app. Does it work with Codespaces? We could have a bug there.

@code-asher have you had any luck?

**Reply by code-asher** [reply]
The debugger should be fine but the simple browser will not work without forwarding the localhost port to the server manually or using the proxy.  https://github.com/coder/code-server/issues/5402
